bokehjs/test/unit/models/widgets/tables/cell_formatters.ts

```diff
@@ -124,21 +124,25 @@ describe("cell_formatters module", () => {
       })
 
       it("should apply to_fixed if value is zero", () => {
-        const df = new ScientificFormatter({precision: 0})
-        expect(df.doFormat(0, 0, 0, {}, {})).to.be.equal('<div class="bk" style="text-align: left;">0</div>')
+        const df = new ScientificFormatter({ precision: 0 })
+        expect(df.doFormat(0, 0, 0, {}, {})).to.be.equal(
+          '<div class="bk" style="text-align: left;">0</div>'
+        )
```

Comment:
On the other hand this is unnecessary, especially space in `{ }`, but splitting the line is also excessive.  We have limited use of a linter in bokehjs, thus you will get feedback regarding basic issues, like indentation, but otherwise the policy is to keep things aligned with the rest of bokehjs.

Reply:
Actually I applied this from Prettier but I'll revert this.
Anyway I feel this is exactly the kind of _time consuming problems_ an autoformatter should solve for you!